packages/apps/docs/src/components/Markdown/Code/style.css.ts

```diff
@@ -78,6 +79,10 @@ export const codeLine = style([
   },
 ]);
 
+globalStyle(`${code} span`, {
+  fontFamily: vars.fonts.$mono,
+})
+
```

Comment:
Since this is a code block, can you just make everything inside fontFamily mono?

Reply:
This bit doesn't work! Despite what's said in the vanilla-extract docs, using `globalStyle` with custom classes doesn't apply them.

The way to get this block to work, and avoid that children and grandchildren stuff, would be:
````typescript
globalStyle(`code span`, {
  fontFamily: vars.fonts.$mono,
})
```

Reply:
hmm strange. As long as this works, I'm ok with it it as well 👍  i think it is still cleaner. What do you think?

Reply:
I agree that's way cleaner, happy to go with it